pom.xml

```diff
@@ -45,6 +46,18 @@
 			<artifactId>spring-security-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>DataServiceJPA</groupId>
```

Comment:
This will only build project on you local machine. Others won't be able to run maven build since we don't use artifact repository to host libraries. Please add jpa related classes to this project.
What was you idea behind this?